Spec/Auth.swift

```diff
@@ -1717,14 +1718,12 @@ class Auth : QuickSpec {
                     waitUntil(timeout: testTimeout) { done in
                         auth.authorise(nil, options: authOptions) { tokenDetails, error in
                             expect(authCallbackHasBeenInvoked).to(beTrue())
-
-                            authCallbackHasBeenInvoked = false
```

Comment:
Because @EvgenyKarkan I believe what @tcard is saying is that you should test that the defaults are reused first (i.e. `Stores the AuthOptions and TokenParams arguments as defaults for subsequent authorisations with the exception of the attributes TokenParams#timestamp and AuthOptions#queryTime`).  The test for http://docs.ably.io/client-lib-development-guide/features/#RSA10g is now missing i.e. we need to ensure that the defaults are carried through to the following request for RSA10g, and for RSA10j (this PR), you need to ensure any provided details override existing defaults.  Added to this, https://github.com/ably/ably-ios/pull/492/files#diff-b5d84fe9bce26902a7481c2ffec84fffR1723 is really not relevant given that param has been removed completely given RSA10d is now removed, see https://github.com/ably/docs/blob/source/content/client-lib-development-guide/versions/features-0-8__0-9.diff#L116.

So in summary:
* Test that defaults are carried through when none are provided (need initial + subsequent request without any arguments)
* Test that defaults are changed when set (need initial + subsequent request without any arguments)
* Remove `force` completely as this just confuses these tests given that option is no longer relevant and should be removed.